### ISSUE TITLE
Improve UX during DocumentServer startup

### ISSUE THREAD

**Issue by dreua** [ONLYOFFICE/DocumentServer#1198]
After a server reboot, the DocumentServer needs significant time to warm up. During this time, you can click "Edit" on documents in HumHub but all you get to see is the overlay window with a big white rectangle as its content. It would be nice to have some information about what is going on and maybe a spinner or something while we wait. (With periodic refresh or refresh once the server is ready.)

**Comment by ShockwaveNN** [comment]
@dreua Hi, thank you for idea, I've moved this issue in DocumentServer base repo, since I think we should implement some kind of message like `DocumentServer is starting up` which will be actual not only for HumHub integration, but for any integration

I've created issue 49356 in our private issue tracker to our team to discuss a possibility of this, but I'm not 100% sure this can be easily implemented for any integration

**Comment by JDSalcedo** [comment]
Commonly the client implements a periodical call to the server like a ping/handshake if the server is on will respond if not then the client should show a message `Trying to connect to server...` to all cases(server down, lost connection, etc...)
